examples/ex1.cpp

```diff
@@ -182,21 +186,41 @@ int main(int argc, char *argv[])
    // 11. Solve the linear system A X = B.
    if (!pa)
    {
+      if (use_ginkgo_solver)
+      {
+#ifdef MFEM_USE_GINKGO
+         // 11b. Solve the linear system A X = B with Ginkgo.
+         std::string executor = "reference";
+         auto exec = gko::ReferenceExecutor::create();
+         auto ilu_precond =
+            gko::preconditioner::Ilu<gko::solver::LowerTrs<>,
+            gko::solver::UpperTrs<>, false>::build()
+            .on(exec);
+         GinkgoWrappers::CGSolver ginkgo_solver(executor, 1, 2000, 1e-6, 0.0,
+                                                ilu_precond.release() );
+         ginkgo_solver.solve(&((SparseMatrix&)(*A)), X, B);
+
```

Comment:
I'm not sure what will be the best way to illustrate the use of Ginkgo, but I don't think we want this change to example 1. Maybe we could have an `examples/ginkgo` directory or add example in the comments of `linalg/gingko.?pp`?

Reply:
I think examples/ginko is a good idea.  We can keep the base examples clean that way, but still test against our integrations that way.

Reply:
If you don't mind go ahead and pull it into it's own example folder like suggested and I will fix up a makefile for it and add it to our sanity checking tests.

Reply:
That is a good idea. I think that will also help when we add more examples, which we probably will soon. I have not yet added it to the `examples/CMakeLists.txt` or the `makefile`. Please feel free to push to this branch and add them as I dont have any more local changes and as I am not really familiar with your testing framework. Thanks!